src/main/resources/assets/admin/common/js/form/FormItemLayer.ts

```diff
@@ -38,6 +38,8 @@ export class FormItemLayer {
 
     private lazyRender: boolean = true;
 
+    private isNew: boolean;
```

Comment:
We should find a better, more explanatory name for this... What does `isNew` stand for in the context of FormItemLayer?